.idea/libraries/sbt__org_apache_bahir_spark_streaming_twitter_2_11_2_2_1_jar.xml

```diff
@@ -0,0 +1,11 @@
+<component name="libraryTable">
```

Comment:
configuration files of a local development environment (e.g. `.idea` folder) should not be published to a CVS.
please exclude them.

Reply:
Fixed...